templates.py

```diff
@@ -54,7 +54,7 @@ def get_tool_descriptions(tools: list[str]) -> str:
  and ends with the following:
 "...{output_end}\""""
 
-notice_retroactively_using_saved_output = """The output of the previous command was long and is being truncated to lower token usage. This message may have initially displayed the entire output, and so subsequent actions may have taken advantage of that information. The full output is saved as {filename}. If you need to refer to it, consider searching that file for the relevant information.
+notice_retroactively_using_saved_output = """The output of the previous command was long and is being truncated to lower token usage. This message may have initially displayed the entire output, and so subsequent actions may have taken advantage of that information. {saved_output_instructions}If you need to refer to it, consider searching that file for the relevant information.
```

Comment:
Nice!

Reply:
Noticed the message still referred to a file in cases where it might not exist. Updated